addons/create_chroot.sh

```diff
@@ -7,23 +7,25 @@ if [ -z "$NS" ] || [ -z "$BASE" ]; then
     exit 1;
 fi
 
-DIRS=(proc var etc lib lib64 usr sbin bin var/cache sys var/lib/samba dev tmp run/samba var/log/samba)
+DIRS=(proc var etc lib lib64 usr sbin bin var/cache/samba sys var/lib/samba dev tmp run/samba var/log/samba var/lock var/run)
```

Comment:
Attempt removing var/lock and var/run to see if all still succeeds as I believe the issue you fixed isn't tied to that